bench_util/Cargo.toml

```diff
@@ -8,10 +8,11 @@ description = "Bench and profiling utilities for deno crates"
 license = "MIT"
 readme = "README.md"
 repository = "https://github.com/denoland/deno"
+publish = false
```

Comment:
I reverted a commit that made usage of `bench_util` a couple days ago. I'm pretty sure we'll have to pubish this crate sooner or later.

Reply:
We'll probably have to rename it before doing that though. I assume we don't want to publish a crate named `bench_util`.